### PR TITLE
Update main.mako full screen

### PR DESCRIPTION
**Pull request by BKSteve**
Full width layout for large screens to match small screens.

Aligns with comments and resolves #7431

### CODE REVIEW

**Comment by miigotu** [comment]
Do you know how to do the if/else in mako and make this a setting in interface settings? Add it in settings.NO_MARGIN or something?

**Comment by BKSteve** [comment]
I do not, yet. 
But I will have a look and try to learn.
It's the ` interface settings` part of the pages that I have 0 experience with so was staying away from it.
Any pointers?

**Comment by miigotu** [comment]
Just make the setting in python first, search for one of the existing settings and see the few places they are set . That will make it an ini only option for now. Then add the mako if/else for the offsets.

**Comment by BKSteve** [comment]
Python settings done and working with a 1 or 0 in GUI section of config.ini

**Comment by miigotu** [comment]
The rest is html inputs and one in save_config or something like that in views

**Comment by BKSteve** [comment]
Yeah, this is the hard bit for me. Zero on where to start or even look.
Thought it would be great to add at the bottom of `Select Columns` list but like a headless chicken no idea where to start.

**Comment by BKSteve** [comment]
Worked on it a little more and learnt some issues and thought best spot was in the General Misc section.
Added the necessary items and cleaned up a little. Working.

Ready for further testing / merge

**Comment by BKSteve** [comment]
bump

File: sickchill/views/config/general.py [hunk 1]
```diff
@@ -154,7 +156,6 @@ def saveGeneral(
         settings.TRASH_ROTATE_LOGS = config.checkbox_to_value(trash_rotate_logs)
         settings.IGNORE_BROKEN_SYMLINKS = config.checkbox_to_value(ignore_broken_symlinks)
         config.change_update_frequency(update_frequency)
-        settings.LAUNCH_BROWSER = config.checkbox_to_value(launch_browser)
```
Comment (miigotu) [inline sickchill/views/config/general.py]: ^^^

File: sickchill/gui/slick/views/layouts/main.mako [hunk 2]
```diff
@@ -324,7 +324,11 @@
                 % endif
             </div>
             <div class="row">
-                <div class="col-lg-10 col-lg-offset-1 col-md-10 col-md-offset-1 col-sm-12 col-xs-12">
+                % if settings.NO_LGMARGIN:
+                    <div class="col-lg-12 col-md-12 col-sm-12 col-xs-12">
+                % else:
+                    <div class="col-lg-10 col-lg-offset-1 col-md-12 col-sm-12 col-xs-12">
```
Comment (miigotu) [inline sickchill/gui/slick/views/layouts/main.mako]: Shouldn't this be the same as before? `"col-lg-10 col-lg-offset-1 col-md-10 col-md-offset-1 col-sm-12 col-xs-12"`

File: sickchill/views/config/general.py (hunk 1 above)
Comment (BKSteve) [reply sickchill/views/config/general.py]: oo that's a bad one to kill.

File: sickchill/gui/slick/views/layouts/main.mako (hunk 2 above)
Comment (BKSteve) [reply sickchill/gui/slick/views/layouts/main.mako]:
```
         <div class="row">
                % if settings.NO_LGMARGIN:
                    <div class="col-lg-12 col-md-12 col-sm-12 col-xs-12">
                % else:
                    <div class="col-lg-10 col-lg-offset-1 col-md-10 col-md-offset-1 col-sm-12 col-xs-12">
```

**Pull request merged**